spec/request/service_credential_bindings_spec.rb

```diff
@@ -1220,6 +1220,14 @@ def check_filtered_bindings(*bindings)
               'request' => create_body.with_indifferent_access
             })
           end
+
+          context 'orphan mitigation' do
```

Comment:
It looks like the shared example does not include all the states that are described in the acceptance criteria scenarios 1, 2 and 4.